pptx_blueprint/__init__.py

```diff
@@ -1,9 +1,14 @@
-import pathlib
+from pathlib import Path
 import pptx
 from typing import Union, Iterable, Tuple
 from pptx.shapes.base import BaseShape
+import subprocess
+import tempfile
+_Pathlike = Union[str, Path]
 
-_Pathlike = Union[str, pathlib.Path]
+
+class LibreOfficeNotFoundError(FileNotFoundError):
```

Comment:
Hmm... I don't really see the advantage of extending `FileNotFoundError`. Looking into the future, I would like to have a common base class for all `pptx_blueprint`-related exceptions, as this would allow for a 'catch all' except clause. And as far as I'm concerned, multiple inheritance is off the table ;-)

Reply:
Not sure I see the point of a 'catch all' specific for all `pptx_blueprint` errors. In particular, we're also not going to warp every possible exception that may be indirectly raised through our code (e.g. cannot save because disk is full).

However, it's ok to just derive from `Exception` here for the moment. We can later decide if we want to refine this.

Reply:
Having a parent exception does only make sense if we would catch all exceptions and rethrow library-specific exceptions, yes. It would allow the end user to write code like this:

```python
try:
    ... do complicated stuff possibly throwing exceptions ...
    pptx_blueprint.save_pdf()
    ... do other complicated stuff possibly throwing exceptions ...
except ComplicatedStuff1Exception as e:
    ... handle errors in complicated stuff no. 1 ...
except pptx_blueprint.ParentException as e:
    ... handle pdf creation errors ...
except Exception:
    ... handle other possible error paths ...
```

This is not a must-have feature, but as an end user I like the convenience of it. Still, you'd have to catch and rethrow all exceptions, and I could perfectly understand anybody arguing that it's just not worth it :)